### PR TITLE
Add SPDX identifier to ERC1820Registry.sol

### LINKED ISSUE #331: License for ERC1820 registry
The file [ERC1820Registry.sol](https://github.com/morpho-org/metamorpho/blob/main/src/mocks/ERC1820Registry.sol) does not come with a SPDX identifier, which raises a warning at compilation.

Additionally, this file mentions the CC0 license which should be packaged along with it: `You should have received a copy of the CC0 Public Domain Dedication along with this software`

### PR DESCRIPTION
**Pull request by Jean-Grimal**
Fixes #331

### CODE REVIEW

**Review by QGarchery — COMMENTED** [review]
Line 15 of this

File: src/mocks/ERC1820Registry.sol
[the remark this answers is not in the document]
Comment (Jean-Grimal) [reply src/mocks/ERC1820Registry.sol:1]: What do you suggest ? adding the license or removing the line ?

[the remark this answers is not in the document]
Comment (QGarchery) [reply src/mocks/ERC1820Registry.sol:1]: Not a lawyer, but from [what I've looked at](https://wiki.creativecommons.org/wiki/CC0_FAQ) it seems like CC0 is compatible with GPL. So maybe just remove the lines ?

[the remark this answers is not in the document]
Comment (Jean-Grimal) [reply src/mocks/ERC1820Registry.sol:1]: I follow your call

**Review by MerlinEgalite — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]

**Comment by MerlinEgalite** [comment]
@QGarchery can you look at the certora failed test pls?

**Review by Rubilmax — APPROVED** [review]

**Review by MathisGD — APPROVED** [review]

**Pull request merged**